lndp_apply_perts.F90

```diff
@@ -15,17 +15,22 @@ module lndp_apply_perts_mod
 !====================================================================
 ! Driver for applying perturbations to sprecified land states or parameters
 ! Draper, July 2020. 
-! Note on location: requires access to namelist_soilveg
+! Note on location: requires access to namelist_soilveg or namelist_soilveg_ruc
```

Comment:
I don't see where the information from `namelist_soilveg_ruc` gets used?

Reply:
Since both `maxsmcnoah` and `maxsmc` are passed in, we do not need to have access to `namelist_soilveg` either, or ?

By the way, what is `maxsmc` when Noah LSM is used? Identical to `maxsmcnoah`?

Reply:
@DomHeinzeller Dom, I am not sure if I made a good PR.   I will remove the old comment line that is not relevant now. And there shouldn't be maxsmcnoah in my latest version. The parameter list is as follows:
subroutine lndp_apply_perts(blksz,lsm, lsoil, lsm_ruc, lsoil_lsm, zs_lsm,         &
                dtf, n_var_lndp, lndp_var_list,                                       &
                lndp_prt_list, sfc_wts, xlon, xlat, stype, smcmax, smcmin,            &
                param_update_flag,                                                    &
                smc, slc, stc, vfrac, alvsf, alnsf, alvwf, alnwf, facsf, facwf,       &
                snoalb, ierr)